lib/health_checks/sidekiq.rb

```diff
@@ -66,5 +83,9 @@ def respond(socket, status, response)
       socket.print "\r\n" # blank line as required by the protocol
       socket.print response
     end
+
+    def log_error(error)
+      Sidekiq::Logging.logger.error "Error: #{error}"
```

Comment:
Again, it's pointless to write `Error: `, that's what log levels are for

Reply:
to have :)